src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,28 @@
+.catalog {
+  display: grid;
+  gap: $gap;
+  grid-template-columns: $cardWidth;
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $cardWidth);
+  }
```

Comment:
You can make a loop to create all media queries. Just create a correctly mapped list to iterate.